geometric_kernels/frontends/gpflow.py

```diff
@@ -33,7 +33,9 @@ def space(self) -> Space:
         return self._kernel.space
 
     def K(self, X, X2=None):
-        return self._kernel.K(X, X2, lengthscale=self.lengthscale)
+        lengthscale = tf.convert_to_tensor(self.lengthscale)
```

Comment:
Why is this necessary? Because we want to extract the type later in the code? If that's the case I think it makes sense to add a global setting.

Reply:
Not only that but also because `self.lengthscale` is a `gpflow.Parameter` which boils down to `tfp.TransformedVariable` and eagerpy does not know how to deal with that.